README.md

```diff
@@ -2,13 +2,412 @@
+
+### Deploy with Local Mode
+
+If your Mattermost server is running locally, you can enable [local mode](https://docs.mattermost.com/administration/mmctl-cli-tool.html#local-mode) to streamline deploying your plugin. After configuring it, just run:
+
+```sh
+make deploy
+```
+
+### Deploy with credentials
+
+Alternatively, you can authenticate with the server's API with a [personal access token](https://docs.mattermost.com/developer/personal-access-tokens.html):
+
+```sh
+export MM_SERVICESETTINGS_SITEURL=http://localhost:8065
+export MM_ADMIN_TOKEN=j44acwd8obn78cdcx7koid4jkr
+make deploy
+```
+
+## Usage
+
+### Slash Commands Overview
+
+- `nagios`
+    - `get-logs <alerts|notifications>`
+        - `[host|service <host name|service description>]`
+    - `set-logs-limit <count>`
+    - `set-logs-start-time <seconds>`
+    - `subscribe <report|configuration-changes>`
+    - `unsubscribe <report|configuration-changes>`
+    - `set-report-frequency <minutes>`
+
+### Slash Commands Documentation
+
+#### nagios
+
+`nagios`
+
+This is the root command.
+
+##### get-logs
+
+`get-logs <alerts|notifications> [<host|service> <host name|service description>]`
+
+This action allows you to get alerts or notifications.
+
+Example: `/nagios get-logs alerts`
+
+###### host
+
+`get-logs <alerts|notifications> host <host name>`
+
+This optional parameter allows you to get alerts or notifications from a specific host.
+
+Example: `/nagios get-logs alerts host bvmhost-p09-02.iad2.fedoraproject.org`
+
+###### service
+
+`get-logs <alerts|notifications> service <service description>`
+
+This optional parameter allows you to get alerts or notifications from a specific service.
+
+Example: `/nagios get-logs alerts service Swap-Is-Low`
+
+##### set-logs-limit
+
+`set-logs-limit <count>`
+
+This action allows you to limit the number of logs `get-logs` fetches.
+
+Example: `/nagios set-logs-limit 10`
+
+##### set-logs-start-time
+
+`set-logs-start-time <seconds>`
+
+This action allows you to specify the age of the oldest log `get-logs` fetches.
+
+Example: `/nagios set-logs-start-time 3600`
+
+##### subscribe
+
+`subscribe <report|configuration-changes>`
+
+This action allows you to subscribe to system monitoring reports or configuration changes on the current channel.
+
+Example: `/nagios subscribe report`
+
+##### unsubscribe
+
+`unsubscribe <report|configuration-changes>`
+
+This action allows you to unsubscribe from system monitoring reports or configuration changes on the current channel.
+
+Example: `/nagios unsubscribe configuration-changes`
+
+##### set-report-frequency
+
+`set-report-frequency <minutes>`
+
+This action allows you to set the frequency of system monitoring reports.
+
+Example: `/nagios set-report-frequency 60`
+
+## Nagios Core API Client
+
+This is a Go client for the Nagios Core API that can communicate with the Nagios Core JSON CGIs. It's the programmatic equivalent of the <nagios>/nagios/jsonquery.html.
+
+### Support
+
+**Archive JSON CGI**
```

Comment:
I'm not sure if this section should be here either

Reply:
If it's no longer applicable to the plugin, let's remove it.

Reply:
Where did this `Nagios Core API Client` content come from? I don't see it in the removed diff

> Is there someone I should add as a reviewer on this PR to identify all content that can be safely omitted from the README page?

The community members that developed the plugin may be able to provide some insight @amwolff & @DanielSz50. @DHaussermann may also have some insight on this

Reply:
The ``Nagios Core API Client`` content was published via GitBook.